ocs_ci/helpers/helpers.py

```diff
@@ -5734,6 +5750,73 @@ def apply_custom_taint_and_toleration(taint_label="xyz"):
                 pod_obj.delete(wait=False)
 
 
+def remove_toleration():
+    """
+    Remove toleration on storagecluster, subscription, configmap, ocsinit and drivers
```

Comment:
I dont see where function handle configMap or OCSInit, plz fix docstring or add the functionality